taskcat/stacker.py

```diff
@@ -1034,7 +1036,7 @@ def generate_input_param_values(self, s_parms, region):
 
                 if licensecontent_re.search(param_value):
                     license_bucket = 'quickstart-ci-license'
-                    licensekey = (self.regxfind(licensecontent_re, param_value)).strip('/',1)[1]
+                    licensekey = '/'.join((self.regxfind(licensecontent_re, param_value)).split('/')[1:])
```

Comment:
should this commit be rolled into this pr ?

Reply:
Its Andrews fix remove it from this Pr